tests/hmm/hmumu_utils.py

```diff
@@ -1541,7 +1620,8 @@ def dnn_variables(leading_muon, subleading_muon, leading_jet, subleading_jet, ns
     #delta eta between jets 
     jj_deta = leading_jet["eta"] - subleading_jet["eta"]
     jj_dphi = leading_jet["phi"] - subleading_jet["phi"]
-    jj_dphi_mod = NUMPY_LIB.mod(jj_dphi + math.pi, math.pi)
```

Comment:
Did you have any issue with `numpy.mod`? The current deltaphi loop is pure python (not accelerated), so the analysis will be bottlenecked by this. You can overcome this by writing it as a numba function, see an example here: https://github.com/jpata/hepaccelerate-cms/blob/dev-aug-w2/tests/hmm/hmumu_utils.py#L1395

Reply:
I think what numpy.mod gives is not how the input variable is defined for the BDT. I will have a look at the numba function. Thanks for pointing this out.

Reply:
A numba function for the correct delta phi has been uploaded.